lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -266,15 +294,16 @@ class LineChartPainter extends AxisChartPainter<LineChartData>
   /// and we use isCurved to find out how we should generate it,
   /// If you want to concatenate paths together for creating an area between
   /// multiple bars for example, you can pass the appendToPath
-  Path _generateBarPath(Size viewSize, LineChartBarData barData, {Path appendToPath}) {
+  Path _generateBarPath(Size viewSize, LineChartBarData barData,
+      {Path appendToPath, List<FlSpot> bar}) {
```

Comment:
Why you added `bar` as an optional parameter?
I think it should not be optional.

Reply:
I'm asking for change this `bar` to `barSpots`.

Reply:
Sorry let me fix

Reply:
done